pkg/handlers/handlers.go

```diff
@@ -0,0 +1,18 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+)
+
+func Spin(http.ResponseWriter, *http.Request) {
+	log.Println("starting")
+	for {
+		for i := 0; i < 100000000; i++ {
+		}
+	}
+}
+
+func Unspin(http.ResponseWriter, *http.Request) {
+	log.Println("stopping")
```

Comment:
how is this stopping the for?

Reply:
welp I just realised my mental model is totally wrong (haven’t worked with this in such a long time!)

will update tomorrow!

Reply:
👍  I think you could do something cute with channels here

Reply:
done :D